account.js

```diff
@@ -112,11 +112,43 @@ class Account {
         }));
     }
 
+    /**
+     * Adds a new unrestricted key to the owners account.
+     *
+     * @param {string} ownersAccountId id of the owner's account.
+     * @param {string} publicKey public key encoded in bs58
+     */
+    async addAccountKey(ownersAccountId, publicKey) {
+        const nonce = await this.nearClient.getNonce(ownersAccountId);
+        publicKey = bs58.decode(publicKey);
+        return this.nearClient.signAndSubmitTransaction(ownersAccountId, AddKeyTransaction.create({
+            nonce,
+            originator: ownersAccountId,
+            newKey: publicKey,
+        }));
+    }
+
+    /**
+     * Removes given unrestricted key from the owners account.
+     *
+     * @param {string} ownersAccountId id of the owner's account.
+     * @param {string} publicKey public key encoded in bs58
+     */
+    async removeAccountKey(ownersAccountId, publicKey) {
+        const nonce = await this.nearClient.getNonce(ownersAccountId);
+        publicKey = bs58.decode(publicKey);
+        return this.nearClient.signAndSubmitTransaction(ownersAccountId, DeleteKeyTransaction.create({
+            nonce,
+            originator: ownersAccountId,
+            curKey: publicKey,
```

Comment:
Either `cur_key` in `removeAccessKey` is incorrect or `curKey` here is incorrect. Since you have a test, it probably the case that `removeAccessKey` is not correctly working.

Reply:
See here https://github.com/nearprotocol/nearlib/pull/37